### PR TITLE
Initial board support

### PR DESCRIPTION
**Pull request by agherzan**
Fixes: https://github.com/resin-os/resinos/issues/198

### CODE REVIEW

File: layers/meta-resin-up-board/recipes-core/images/resin-image-flasher.bbappend [hunk 1]
```diff
@@ -0,0 +1,7 @@
+include resin-image.inc
+
+# Put the initramfs inside the boot partition
```
Comment (floion) [inline layers/meta-resin-up-board/recipes-core/images/resin-image-flasher.bbappend]: so where do you put the initramfs inside the boot partition? I see you only copy gfub config files here

File: up-board.coffee [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+		osx: 'http://docs.resin.io/#/pages/installing/gettingStarted-UPboard.md#on-mac-and-linux'
+		linux: 'http://docs.resin.io/#/pages/installing/gettingStarted-UPboard.md#on-mac-and-linux'
+
+	supportsBlink: true
+
+	yocto:
+		machine: 'up-board'
+		image: 'resin-image-flasher'
+		fstype: 'resin-sdcard'
+		version: 'yocto-jethro'
```
Comment (floion) [inline up-board.coffee]: this should probably be yocto-krogoth?

File: up-board.coffee [hunk 3]
```diff
@@ -0,0 +1,57 @@
+deviceTypesCommon = require '@resin.io/device-types/common'
+{ networkOptions, commonImg, instructions } = deviceTypesCommon
+
+UPBOARD_FLASH = '''
+	Power up the <%= TYPE_NAME %>. Make sure you have a keyboard connected.
+	Press the F7 key while BIOS is loading in order to enter the boot menu. Next, select the "UEFI:" option.
+'''
+
+UPBOARD_POWERRESET = 'Plug and unplug power on you <%= TYPE_NAME %>.'
```
Comment (floion) [inline up-board.coffee]: This should probably be:

Plug and unplug power on **your**

Comment (floion) [inline up-board.coffee]: And should it not be the other way around? Like: unlug and plug? Instead of plug and unplug?

Comment (floion) [inline up-board.coffee]: Also, how does this board powers on? Does it have a button you need to push? Or it just boots when power plug is connected?

Comment (agherzan) [inline up-board.coffee]: I'll fix the typos. It boots at plug - not button.

Comment (agherzan) [inline up-board.coffee]: Fixed

File: layers/meta-resin-up-board/recipes-core/images/resin-image-flasher.bbappend (hunk 1 above)
Comment (agherzan) [inline layers/meta-resin-up-board/recipes-core/images/resin-image-flasher.bbappend]: Removed altogether.

**Comment by agherzan** [comment]
@floion indeed. In 10 days we got only one review. If what you say is that this is worrying then I 100% agree.

**Pull request merged**